ci/gpu/java.sh

```diff
@@ -88,8 +88,8 @@ gpuci_conda_retry install -y \
                   "maven"
 
 # https://docs.rapids.ai/maintainers/depmgmt/
-# gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
-# gpuci_conda_retry install -y "your-pkg=1.0.0"
+# gpuci_mamba_retry remove --force rapids-build-env rapids-notebook-env
```

Comment:
FYI, we can't use `mamba remove` with the `--force` flag. It currently has a bug. You can read more about it in the doc section below.

https://docs.rapids.ai/maintainers/depmgmt/#testing-a-dependency

![image](https://user-images.githubusercontent.com/7400326/148455899-f0d1762b-160a-4d26-9cb0-a90b9a318f81.png)

Reply:
@ajschmidt8 Thank you for this info! I just saw that was changed in PR #9486 (https://github.com/rapidsai/cudf/pull/9486/files#r773465635) but I didn't know why.

Reply:
```suggestion
# gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
```